src/controller/python/chip/yaml/format_converter.py

```diff
@@ -60,7 +60,7 @@ def convert_name_value_pair_to_dict(arg_values):
     return ret_value
 
 
-def convert_yaml_type(field_value, field_type, use_from_dict=False):
+def convert_yaml_type(field_value, field_type, config_values, use_from_dict=False):
```

Comment:
What does "use_from_dict" do?

Please give basic list of arguments and their meaning in the docstring, since it's a very non-obvious method

Reply:
I will address this in a follow up PR where we more properly implement using config values (as we discussed in the call), since we are not making any modifications to this file in this PR